[Doc] Some instructions on how to size the head node (#36429)
ray-project/ray: doc/source/cluster/vms/user-guides/large-cluster-best-practices.rst

^^^^^^^^^^^^^^^^^^^^^^^^^

In addition to the above changes, when deploying a large cluster, Ray's

> architecture means that the head node will have extra stress due to

Reviewer: ```suggestion
architecture means that the head node has extra stress due to
```

additional system processes running on it like GCS.

* A good starting hardware specification for the head node is 8 CPUs and 32 GB memory.
  The actual hardware specification depends on the workload and the size of the cluster.
  Metrics that are useful for deciding the hardware specification are

>   CPU usage, memory usage and network bandwidth usage.

Reviewer: ```suggestion
  CPU usage, memory usage, and network bandwidth usage.
```
